release-notes/1.19.2.md

```diff
@@ -0,0 +1,26 @@
+## Introduction
+
+The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.19.2.
+
+Please see [component versions](https://anbox-cloud.io/docs/ref/component-versions) for a list of updated components.
+
+## New features & improvements
+
+* When the Anbox Cloud Appliance upgrade fails, you can force run the upgrade again by using the `--force` option when running `anbox-cloud-appliance upgrade`.
+* Android security updates for October 2023 (see [Android Security Bulletin - October 2023](tbd) for more information).
+* The Android WebView has been updated to [tbd](tbd).
+
+## Bugs fixed
+* Screen rotation fails to work and causes an `Unknown orientation type` error. <!--AC-1864-->
+* The existing telegraf configuration that collects metrics for the running containers does not work for cgroup-v2. <!--AC-1919-->
+* Anbox Cloud Appliance fails to initialise with the following error on systems that have EFI but do not support secure boot: <!--AC-1951-->
+    `Error: failed to check secure boot status: exit status 255`
+* A container is deleted when its start operation fails. <!--AC-1952-->
+* After a container is stopped, GPUs and VPUs assigned to it are not cleared. This leads to having multiple GPUs/VPUs assigned to the container without proper allocation or utilisation. <!--AC-1953-->
+* When a VPU based on NVMe is assigned to a container, the corresponding control device is not assigned. This could prevent the drivers from using the control device. <!--AC-1955-->
```

Comment:
```suggestion
* When a NVMe-based VPU is assigned to a container, the corresponding control device is not assigned. This could prevent the drivers from using the control device. <!--AC-1955-->
```
What do you think?